.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
added prettier to get formatting issues out of the way

Reply:
Good idea. Not super important but I wonder if we should also add these settings (used in the main repo):
```
  printWidth: 120, // or 100? 
  tabWidth: 2,
  trailingComma: 'all',
```

Reply:
We can adjust these things once we migrate this to official CRA template under @fluentui repo.